pkg/worker/config.go

```diff
@@ -47,6 +47,7 @@ func LoadConfig() (*plugin.WorkerConfig, error) {
 	viper.BindEnv("masterurl", "MASTER_URL")
 	viper.BindEnv("nodename", "NODE_NAME")
 	viper.BindEnv("resultsdir", "RESULTS_DIR")
+	viper.BindEnv("resulttype", "RESULT_TYPE")
```

Comment:
Look into this one.

Reply:
We do need this. I am providing these values to the worker via Env Vars in the plugin template. This allows me to bypass the ConfigMap entirely.